### PR TITLE
Fix path conflicts

### PR DESCRIPTION
**Pull request by lambdaFun94**

### CODE REVIEW

File: src/templates/location.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * is generated for every corresponding (based on the filter) stream document.
  *
  * Another way to think about it is that a page will be generated using this
- * template for every eligible entity in your Knowledge Graph.
+ * template for every eligible entity in your Knowledge Graph
```
Comment (mkilpatrick) [inline src/templates/location.tsx:8]: Any reason you removed the periods from these sentences?

File: src/templates/robots.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ export const getPath = () => {
 
 export const render = (): string => {
   /*
-   * Return a string that will be served at robots.txt.
+   * Return a string that will be served at robots.txt
```
Comment (mkilpatrick) [inline src/templates/robots.ts:18]: And here?

File: src/templates/location.tsx (hunk 1 above)
Comment (lambdaFun94) [reply src/templates/location.tsx:8]: I didn't do that. Maybe it was the formatter?!

Comment (lambdaFun94) [reply src/templates/location.tsx:8]: Oh I know why!! It was to trigger a rebuild when I was doing the fleet deploys...lol

File: src/templates/robots.ts (hunk 2 above)
Comment (lambdaFun94) [reply src/templates/robots.ts:18]: same thing here

**Review by mkilpatrick — APPROVED** [review]

**Pull request merged**